crates/biome_resolver/src/resolver_fs_proxy.rs

```diff
@@ -0,0 +1,124 @@
+use biome_fs::{FileSystem, FsErrorKind, PathKind, expand_symbolic_link};
+use biome_package::{Manifest, PackageJson, TsConfigJson};
+use camino::{Utf8Path, Utf8PathBuf};
+
+use crate::errors::ResolveError;
+
+/// Represents the kind of filesystem entry a path points at.
+#[derive(Clone, Debug)]
+pub enum PathInfo {
+    Directory,
+    File,
+    Symlink {
+        /// The normalized target of the symlink.
+        ///
+        /// Accessing this target is guaranteed to not return another symlink.
+        normalized_target: Utf8PathBuf,
```

Comment:
Should we call it `expanded_target` instead? The term "normalise" referred to paths is always about slashes

Reply:
Oh, I can use `canonicalized_target`.